com.unity.render-pipelines.universal/Runtime/ShaderUtils.cs

```diff
@@ -53,5 +53,31 @@ public static bool IsLWShader(Shader shader)
         {
             return s_ShaderPaths.Contains(shader.name);
         }
+
+#if UNITY_EDITOR
+        static readonly string[] s_ShaderGUIDs =
+        {
+            "933532a4fcc9baf4fa0491de14d08ed7",
+            "8d2bb70cbf9db8d4da26e15b26e74248",
+            "650dd9526735d5b46b79224bc6e94025",
+            "69c1f799e772cb6438f56c23efccb782",
+            "b7839dad95683814aa64166edc107ae2",
+            "8516d7a69675844a7a0b7095af7c46af",
+            "0406db5a14f94604a8c57ccfbc9f3b46t",
+            "0ca6dca7396eb48e5849247ffd444914",
+        };
+
+        public static string GetShaderGUID(ShaderPathID id)
```

Comment:
Should we make this internal? Otherwise we need public API docs.

Reply:
Yes, I made that internal in latest commit.